src/index.js

```diff
@@ -1,13 +1,13 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
-import App from './App';
+import Calculator from './components/calculator';
 import reportWebVitals from './reportWebVitals';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
-    <App />
+    <Calculator />
```

Comment:
- [ ] Please make sure to make the wrapper of the Calculator component is the App component. 🙏 Also, make sure that the index.js file is only to wrap the Parent Component 'App.js'

Reply:
Right, I had taken a shortcut 😆 Thanks @MahmoudBakr23